core/src/main/java/com/klaytn/caver/utils/Utils.java

```diff
@@ -0,0 +1,87 @@
+package com.klaytn.caver.utils;
+
+import org.bouncycastle.math.ec.ECPoint;
+import org.web3j.crypto.Hash;
+import org.web3j.crypto.Sign;
+import org.web3j.utils.Numeric;
+
+import java.util.regex.Pattern;
+
+public class Utils {
+    public static final int LENGTH_ADDRESS_String = 40;
+    public static final int LENGTH_PRIVATE_KEY_STRING = 64;
+    private static final Pattern HEX_STRING = Pattern.compile("^[0-9A-Fa-f]+$");
+
+    public static boolean isPrivateKeyValid(String privateKey) {
+        String noHexPrefixKey = Numeric.cleanHexPrefix(privateKey);
+        if(noHexPrefixKey.length() != LENGTH_PRIVATE_KEY_STRING && HEX_STRING.matcher(noHexPrefixKey).matches()) {
+            return false;
+        }
+
+        ECPoint point = Sign.publicPointFromPrivate(Numeric.toBigInt(privateKey));
+        return point.isValid();
+    }
+
+    public static boolean isValidAddress(String input) {
```

Comment:
Also how about changing to `isAddress`? 
caver-js already support those utils functions, so i think support same function with same name is better.

Reply:
ok, I will modify it.